helpers/markdownDocumentsReader.ts

```diff
@@ -17,12 +19,21 @@ export function getPostDocumentBySlug(slug: string) {
 
   return JSONSerialize({ slug: realSlug, ...data, content }) as PostDocument;
 }
-
 export function getAllPostDocuments(): PostDocumentWithoutContent[] {
   const slugs = fs.readdirSync(documentsDirectory);
   const docs = slugs
     .map((slug) => getPostDocumentBySlug(slug))
-    .filter((post: PostDocument) => post.draft === false)
+    .filter((post: PostDocument) => {
+      {
+        if (process.env.NODE_ENV === "development") {
```

Comment:
Add a const on the top of the function

const isDev = process.env.NODE_ENV === "development"

And then check for isDev only

Reply:
@ntorbinskiy